viewing_party/party.py

```
@@ -1,23 +1,121 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+    if title and genre and rating:
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
+    else:
+        return 
+
+def add_to_watched(user_data, movie):
+    if not movie:
+        return user_data

```

We didn't have a particular specification for this situation, but you have elected to silently ignore any falsy values for `movie`, which works fine for the tests. It's definintely true that we wouldn't want to add this to our list, and so it would become a question of whether this is a situation we should silently ignore, or should we raise a racket about it (ar at least an error 😁).